src/compiler/scala/tools/nsc/ast/parser/Parsers.scala

```diff
@@ -1311,8 +1311,13 @@ self =>
       def finish(value: Any): Tree = try newLiteral(value) finally in.nextToken()
       if (in.token == INTERPOLATIONID)
         interpolatedString(inPattern = inPattern)
-      else if (in.token == SYMBOLLIT)
+      else if (in.token == SYMBOLLIT) {
+        def msg(what: String) =
+          s"""symbol literal is $what; use Symbol("${in.strVal}") instead"""
```

Comment:
you can recommend the `sym` interpolator

Reply:
Done in 579c620